scripts/cangen/README.md

```diff
@@ -0,0 +1,36 @@
+# Cangen
+Cangen is a cli package used to generate C++/C files from a corresponding dbc file. This tool automates the can generation process and ensures can generation is consistent between all projects. 
+
+### Installation
+First install the cangen package through the terminal:
+
+```
+pip install racecar/scripts/cangen
```

Comment:
Is there a path I need to run this from?

Reply:
This command installs into the current python virtual environment (or global environment if no venv is activated) and it just needs the path to the `cangen` folder. You could run `pip install cangen` from within `racecar/scripts` or even `pip install ../../scripts/cangen` from inside `racecar/firmware/mcal`.

We should update the docs to install `scripts/cangen` and tell the user to run it from the `racecar/` directory

Reply:
I updated the readme and recommitted the code.